src/extensions/default/bramble/lib/MouseManagerRemote.js

```diff
@@ -1,6 +1,21 @@
 /*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, forin: true, maxerr: 50, regexp: true, bitwise: true */
 /* global addEventListener, removeEventListener, sessionStorage */
-(function(transport) {
+
+var testSessionStorage = "testSessionStorage";
+var noop = function() {};
+var sessionStorage = {
+  getItem: noop,
+  setItem: noop
+};
+
+try {
```

Comment:
Let's add a comment above this that indicates that we're doing this in order to deal with security issues in browsers that don't allow third-party frames to use storage.